### PR TITLE
Extend Web/CSS/Class_selectors

### PR DESCRIPTION
**Pull request by WinterSilence**
#### Summary

Added example discovering how merges properties of classes declared in attribute "class".

#### Motivation

This helps to juniors avoid a popular mistake.

#### Metadata

- [ ] Adds a new document
- [x] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/css/class_selectors/index.md [hunk 1]
```diff
@@ -70,6 +74,13 @@ Note that this is equivalent to the following {{Cssxref("Attribute_selectors", "
 <p>This is just a regular paragraph.</p>
 ```
 
+If the property is declared in several classes of element, then using value of the last declared class in CSS. The order of the class names in the "class" attribute is not important in this case.
```
Comment (wbamberg) [inline files/en-us/web/css/class_selectors/index.md:77]:
```suggestion
If an element a `class` attribute containing multiple classes, and the CSS has conflicting rules for each class, then the rule that is applied is the last rule given in the CSS. The order of the classes in the `class` attribute is not important in this case.
```

something like this is clearer?

File: files/en-us/web/css/class_selectors/index.md [hunk 2]
```diff
@@ -70,6 +74,13 @@ Note that this is equivalent to the following {{Cssxref("Attribute_selectors", "
...
+
+```html
+<p class="blue red">This paragraph has blue text.</p>
+<p class="red blue">This paragraph has blue text.</p>
+```
+
 ### Result
 
 {{EmbedLiveSample('Examples')}}
```
Comment (wbamberg) [inline files/en-us/web/css/class_selectors/index.md:86]:
```suggestion
{{EmbedLiveSample('Examples', '600', '260')}}
```

(let's make the example iframe a bit taller, so we can see these new lines of output)

**Review by wbamberg — CHANGES_REQUESTED** [review]
Thanks for your PR, @WinterSilence ! I had a couple of suggestions.

File: files/en-us/web/css/class_selectors/index.md (hunk 1 above)
Comment (WinterSilence) [reply files/en-us/web/css/class_selectors/index.md:77]: @wbamberg this doc using term "style properties" instead "rules"

Comment (wbamberg) [reply files/en-us/web/css/class_selectors/index.md:77]: I think "conflicting rules" is more comprehensible here, but if you have an alternative that seems better I'm happy to hear it.

Comment (WinterSilence) [reply files/en-us/web/css/class_selectors/index.md]: @wbamberg me too, but not here - need create new  PR and replace "style properties" to "rules" globally to not confuse readers.

File: files/en-us/web/css/class_selectors/index.md [hunk 3]
```diff
@@ -70,9 +74,16 @@ Note that this is equivalent to the following {{Cssxref("Attribute_selectors", "
 <p>This is just a regular paragraph.</p>
 ```
 
+If an element a `class` attribute containing several classes, and they defines same property/rule, then his value set by latest class declared in the CSS. The order of the class names in the `class` attribute is not important in this case.
```
Comment (wbamberg) [inline files/en-us/web/css/class_selectors/index.md:77]: Again, this sentence is very hard to understand. I do think my original suggestion is fine here.

Comment (WinterSilence) [reply files/en-us/web/css/class_selectors/index.md:77]: @wbamberg your version not correct:

> conflicting rules

it's not conflict

> each class

not each

>  very hard to understand

Yes. therefore, the text is provided with a simple example.

Comment (estelle) [inline files/en-us/web/css/class_selectors/index.md:77]:
```suggestion
The order of the class names in a `class` attribute containing multiple space-separated classes has no impact on [specificity](/en-US/docs/Web/CSS/Specificity) or the [cascade](/en-US/docs/Web/CSS/Cascade).
```

Comment (estelle) [reply files/en-us/web/css/class_selectors/index.md:77]: hi @WinterSilence. Feel free to commit my suggested rewrite if that is what you were trying to say.

Comment (WinterSilence) [reply files/en-us/web/css/class_selectors/index.md:77]: @estelle it's too not true - class order impact on specificity, but cascade have high priority

**Pull request closed without merging**